src/components/LiquidityMining/Create/Steps/PairAndReward/index.tsx

```diff
@@ -1,101 +1,223 @@
+    else return 0
+  }, [stakingCapString, inputRef])
+  const handleLocalStakingCapChange = useCallback(
+    rawValue => {
+      if (!stakeTokenOrPair || (stakeTokenOrPair instanceof Pair && !stakeTokenOrPair.liquidityToken)) return
+      setStakingCapString(rawValue)
+      const tokenOrPair = stakeTokenOrPair instanceof Token ? stakeTokenOrPair : stakeTokenOrPair.liquidityToken
+      const parsedAmount = tryParseAmount(rawValue, tokenOrPair) as TokenAmount | undefined
+      onStakingCapChange(parsedAmount || new TokenAmount(tokenOrPair, '0'))
     },
-    [onRewardTokenChange]
+    [onStakingCapChange, stakeTokenOrPair]
   )
 
   return (
     <>
-      <Card>
-        <FlexContainer justifyContent="stretch" width="100%">
-          <Box flex="1">
-            <AssetSelector
-              title="LIQUIDITY PAIR"
-              currency0={liquidityPair?.token0}
-              currency1={liquidityPair?.token1}
-              onClick={handleOpenPairSearch}
-            />
-          </Box>
-          <Box mx="18px">
-            <Divider />
-          </Box>
-          <RewardTokenContainer flex="1">
-            <AssetSelector title="REWARD TOKEN" currency0={reward?.token} onClick={handleOpenCurrencySearch} />
-          </RewardTokenContainer>
-        </FlexContainer>
-      </Card>
+      <FlexContainer marginTop={'32px'}>
+        <AssetSelector
+          campaingType={campaingType}
+          currency0={
+            stakeTokenOrPair && stakeTokenOrPair instanceof Token ? stakeTokenOrPair : stakeTokenOrPair?.token0
+          }
+          currency1={stakeTokenOrPair && stakeTokenOrPair instanceof Token ? null : stakeTokenOrPair?.token1}
+          onClick={() => handelOpenPairOrTokenSearch(campaingType)}
+        />
+
+        <SmoothGradientCard
+          justifyContent={'space-between'}
+          flexDirection={'column'}
+          padding={'33.45px 41px'}
+          height="150px"
+          width="fit-content"
+          disabled={!stakeTokenOrPair}
+        >
+          <TYPE.mediumHeader
+            alignSelf={'start'}
+            letterSpacing="0.08em"
+            color="text3"
+            fontSize={13}
+            fontWeight="600"
+            lineHeight="22px"
+          >
+            MAX STAKED
+          </TYPE.mediumHeader>
+          <FlexContainer marginTop="16px" flexDirection={'row'} width={'100%'} justifyContent={'space-between '}>
+            <StyledFlex
+              marginRight="20px"
+              onClick={() => onUnlimitedPoolChange(true)}
+              alignItems={'center'}
+              active={unlimitedPool}
+            >
+              <StyledUnlimitedText active={unlimitedPool}>UNLIMITED</StyledUnlimitedText>
+            </StyledFlex>
+            <AmountFlex onClick={() => onUnlimitedPoolChange(false)}>
+              <BorderContainer active={!unlimitedPool}>
+                <StyledNumericalInput
+                  style={{ width: widthValue + 12 + 'px' }}
```

Comment:
Can we end this calculation inside the `widthValue` and pass it as a prop to the styled component?

Reply:
yeah I fixed it here also in simulate pr I think it will be fine once we merge it